understanding/20/focus-order.html

```diff
@@ -55,6 +53,28 @@ <h2>Intent of Focus Order</h2>
          sets of users and does not appear to either of them to jump around randomly. 
       </p>
       
+      <p>Focus order also does not need to follow the visual presentation of the web page,
+         as long as the order in which elements receive focus is logical, and the hierarchy and relationship
+         of content implied by the visual presentation is preserved. For instance, in a two-column
+         layout, visually the assumption in western left-to-right systems may be that elements in the
```

Comment:
Given the preceding sentence, I don't think "visually" is needed here, and it's making the sentence a bit harder to read
```suggestion
         layout, the assumption in western left-to-right systems may be that elements in the
```

Reply:
fairy snuff